tests/BrowserFactoryTest.php

```diff
@@ -26,7 +26,7 @@ public function testBrowserFactory()
 
         $browser = $factory->createBrowser();
 
-        $this->assertRegExp('#^ws://#', $browser->getSocketUri());
+        $this->assertMatchesRegularExpression('#^ws://#', $browser->getSocketUri());
```

Comment:
Please don;t change this.

Reply:
`assertRegExp` is deprecated. Why not to change it?

Reply:
Because it won;t work on php 7.1.

Reply:
I checked PHPUnit's code and I did not find anything that would make `assertMatchesRegularExpression` fail on PHP 7.1.

I'm having some docker problems so I'm not being able to reproduce the environment. Because of that I'll revert this change, but I still don't understand what would break on PHP 7.1